src/api/top-stories/queries.ts

```diff
@@ -16,3 +17,18 @@ export const prefetchTopStories = async (queryClient: QueryClient) =>
     queryKey: [QueryKeys.TOP_STORIES],
     queryFn: fetchTopStories,
   });
+
+export const useFetchSingleStory = (slug: string) =>
+  useQuery({
+    queryKey: [QueryKeys.TOP_SINGLE_STORY],
+    queryFn: () => fetchTopStoriesBySlug(slug),
+  });
+
+export const prefetchSingleStory = async (
+  queryClient: QueryClient,
+  slug: string
+) =>
+  await queryClient.prefetchQuery({
```

Comment:
do we need await here?

Reply:
No, that's my fault, it's not necessary in this case because we already marked the function async. It might be useful in a case where I wanna handle errors